tests/lib/options.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @fileoverview Tests for options.
+ * @author Aniketh Saha
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const assert = require("assert"),
+    { normalizeOptions } = require("../../lib/options");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+describe("options unit testing", () => {
```

Comment:
Can we change this to the following, since we're only testing the one public method of this module at the moment?

```suggestion
describe("normalizeOptions", () => {
```